docs/public/content/admin/tutorials/registry.md

```diff
@@ -0,0 +1,49 @@
+# Push Docker images on a registry
+
+!!! warning "Official @@PROJECT_NAME@@ images"
+    @@PROJECT_NAME@@ does **not** officially publish any docker image on any public registry such as Docker hub or Quay.io. That said it provides a Bash script to push local images to any Docker registry.
```

Comment:
@@PROJECT_NAME@@ does **not** officially publish Docker images on any public registry such as Docker Hub or Quay.io. The provided Bash script is meant to push images to **local** Docker repositories running within your organization.